tensorflow/stream_executor/dnn.h

```diff
@@ -667,8 +667,33 @@ class PoolingDescriptor {
   std::vector<int64> strides_;
 };
 
-typedef int64 AlgorithmType;
-constexpr AlgorithmType kDefaultAlgorithm = -1;
+class AlgorithmType {
+  private:
```

Comment:
The convention is that the data fields are at the end.

Reply:
Done.